zerver/openapi/zulip.yaml

```diff
@@ -19004,6 +19019,12 @@ paths:
                               be hidden in all Zulip clients.
 
                               **Changes**: New in Zulip 10.0 (feature level 350).
+                          web_left_sidebar_unreads_count_summary:
+                            type: boolean
+                            description: |
+                              Determines whether the web/desktop application's left sidebar displays the unread message count summary.
+
+                              **Changes**: New in Zulip 10.0 (feature level 354).
```

Comment:
These numbers are stale. You can use `tools/create-api-changelog` to create an ID that can be automatically renumbered when I merge the PR and then use that here.